### ISSUE TITLE
Sidebar is overlapping content of file picker ("Save as" / "Open")

### ISSUE THREAD

**Issue by JammingBen** [owncloud/web#11875]
## Describe the bug
The right sidebar is overlapping the content of the file picker ("Save as" / "Open" actions) if opened.

## Steps to reproduce
1. Open a text file
2. Open the right sidebar
3. From the context menu, click "Save as" or "Open"

## Expected behavior
The sidebar should not be visible in the file picker.

## Actual behavior
It's visible and overlapping all the content.

**Comment by AlexAndBear** [comment]
Now, that I fixed the modal width is this issue still relevant ?

**Comment by AlexAndBear** [comment]
Got you. As of now I can't think of a proper solution as the iframe and the window so share the same local storage

**Comment by JammingBen** [comment]
True, hmmm. All solutions I can think of are somewhat hacky... Maybe just ignore the sidebar storage setting in embed mode and use a local state in the `useSideBar` composable instead? Would be sufficient for an embedded Web I guess.

**Comment by AlexAndBear** [comment]
I am not 100% sure about that, @kulmann already spoke about it in the reviews and decided to leave it as it is. I suggest to give it p4, now we have the width fixed and decide on a solution later

**Comment by JammingBen** [comment]
Fine be me!
